csi/csi.go

```diff
@@ -37,6 +37,7 @@ type OsdCsiServerConfig struct {
 // OsdCsiServer is a OSD CSI compliant server which
 // proxies CSI requests for a single specific driver
 type OsdCsiServer struct {
+	Server
```

Comment:
Just curious, why do we need this? It should work without adding the nested interface as well.